### ISSUE TITLE
GeoIP Database is no longer available for public download

### ISSUE THREAD

**Issue by gerhard-strauss-jambit** [cydrobolt/polr#540]
## Expected Behavior
Executing "php artisan geoip:update" downloads an Update for GeoIP DB

## Current Behavior
It fails
```
php artisan geoip:update
Updating...

  [ErrorException]
  get_headers(): php_network_getaddresses: getaddrinfo failed: Name or service not known

```
## Possible Solution
https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/
The Vendor now demands an Account and a License to download the DB. This needs additional Implementation or a different Vendor

**Comment by gerhard-strauss-jambit** [comment]
As a Workaround, you can do the following:
1. Create MaxMind Account (it is free) and obtain a License Key (also free)
2. Follow their Instructions to create a Direct Download Link -> https://dev.maxmind.com/geoip/geoipupdate/#Direct_Downloads
3. Insert this Link into "config/geoip.php" as "update_url" inside the services -> maxmind_database Hash

**Comment by cydrobolt** [comment]
Could you make a PR to fix this issue?

**Comment by gerhard-strauss-jambit** [comment]
The Workaround is no universal Solution, because the License Key is individual.
If there is no other Solution (Code), maybe it should be mentioned in the Documentation

**Comment by cydrobolt** [comment]
Oops, didn't look to closely! I haven't had time to work on Polr for a while, but I'll take a look at this and mention it in the documentation.

**Comment by est-it** [comment]
Keep in mind, that you become a compressed tar.gz-Archive with the mmbd in it. I have to extract that mmdb first, otherwise GeoIP doesn't work.

**Comment by GomoDD** [comment]
@est-it could you please provide more info about that? I added my license key in the geoip.php "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-City&license_key=YOUR_LICENSE_KEY&suffix=tar.gz" and successfully ran "php artisan geoip:update" is that everything that needs to be done? because I don't think that it's working

**Comment by nikesuputra** [comment]
@GomoDD Hey there! I also got the same problem previously and managed to make it works on my site. I downloaded the tar.gz file to my local computer and extracted it. Then, I uploaded it to my website directory (domain.tld/geoip.mmbd) and I used this link in my config/geoip.php. After that, I ran the `php artisan geoip:update` and it is working fine :smile: 
![image](https://user-images.githubusercontent.com/63705243/90053875-45921d00-dd05-11ea-8c97-9ddaed04dc57.png)
